pcb/rules/F9_1.py

```diff
@@ -49,7 +49,7 @@ def check(self):
             self.error("footprint name (in file) was '{0}', but expected (from filename) '{1}'.\n".format(module.name, os.path.splitext(os.path.basename(module.filename))[0]))
             err = True
 
-        if module.value['value'] != module.name:
+        if str(module.value['value']) != module.name:
             self.error("Value label '{lbl}' does not match filename '{fn}'".format(
                 lbl=module.value['value'],
```

Comment:
same as above.

Reply:
For example Molex uses numeric-only part numbers.

![image](https://user-images.githubusercontent.com/14878875/93082980-81205e00-f69a-11ea-83d0-038874340c2c.png)

You can see the error message below if the module name contains only numeric characters:

```sh
./check_kicad_mod.py ~/work/sharkrf-kicad-library/footprints/srf_Molex.pretty/5051100892.kicad_mod
```
```python
Traceback (most recent call last):
  File "./check_kicad_mod.py", line 117, in <module>
    rule.check()
  File "/home/makos/work/kicad-library-utils/pcb/rules/F7_4.py", line 140, in check
    return any([self.checkPads(module.pads)])
  File "/home/makos/work/kicad-library-utils/pcb/rules/F7_4.py", line 46, in checkPads
    exposed_pad_search = re.search('\-\d+[EP]{2}', fpName, re.IGNORECASE)
  File "/usr/lib64/python3.8/re.py", line 201, in search
    return _compile(pattern, flags).search(string)
TypeError: expected string or bytes-like object
```